ondewo/nlu/agent.proto

```diff
@@ -628,6 +638,33 @@ message ExportAgentResponse {
     }
 }
 
+message ExportBenchmarkAgentRequest {
+    // Agent related info
+    // Required. The project that the agent to export is associated with.
+    // Format: `projects/<Project ID>`.
+    string parent = 1;
+    // Optional. The compression level of the zip file created.
+    // Valid range 1-9; the grpc-default value 0 will be mapped to 1,
+    // other values cause an error in the request validation handler.
+    int32 compression_level = 2;
+
+    // Train-test split related parameters, for further info,
+    // check https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html
+    // Optional: bigger than zero
+    float test_size = 3;
```

Comment:
if not passed, what is the default?

Reply:
@atis06 check this out

Reply:
I see, can you just mention the the default value comes from that API definition?

Reply:
@atis06